src/text-input/src/TextInput.js

```diff
@@ -12,33 +11,30 @@ const TextInput = memo(
       required,
       placeholder,
       width = 280,
-      height = 32,
```

Comment:
Will this work for us in app where we specify height in many places?

Reply:
I believe this will break a ton of the app that specify a height or rely on the default assignment here.

Reply:
look at line `23`

Reply:
Followed the pattern that you did for buttons

Reply:
So i think this should still work fine. There is now a size prop that has the correct default height. And if someone passes their own height that should get passed through with restprops